v0/destinations/snapchat_conversion/transform.js

```diff
@@ -280,7 +333,21 @@ function process(event) {
   let response;
   switch (messageType) {
     case EventType.TRACK:
-      response = trackResponseBuilder(message, destination);
+      const mappedEvents = eventMappingHandler(message, destination);
+      if (mappedEvents.length > 0) {
+        response = [];
+        mappedEvents.forEach(event => {
```

Comment:
`mappedEvents.forEach( mappedEvent => ...`
lets change variable name to meaningful one